### DISCUSSION TITLE
Optional warning dialog when modifying saved views?

### DISCUSSION THREAD

**Discussion by RoL0NL** [paperless-ngx/paperless-ngx#2679] [Feature Requests] [answered]
In #1868, a feature was added (or as stated, a bug was resolved) that allows editing of saved views directly from the view itself: If you 'open' a saved view and change the filters, an '*' is added to its name to indicate unsaved changes, and if you 'leave' the view by opening another view or page, you will get a notification reminding you of these unsaved changed and asks if you want to save them.

While I see that this feature can be helpful to users, it kind of interferes with my workflow: I mainly use the saved views as a starting point for searching through my documents. Often I adjust the filters to narrow down the search and find the document that I am looking for. Now, I constantly get warning dialogs asking if I want to save this view. Would it be possible to make this dialog optional via the settings, and if the dialog is disabled, ignore the changes as it was before this PR?

EDIT: Oops, completely missed this request while searching for existing discussions. Apologies!

**Comment by shamoon** [comment, marked as answer]
Duplicate of #2162
